dapp/src/components/TransactionModal/ActiveStakingStep/DepositBTCModal.tsx

```diff
@@ -36,81 +34,68 @@ export default function DepositBTCModal() {
   }, [dispatch, handleBitcoinBalanceInvalidation])
 
   const onError = useCallback(
-    (error?: unknown) => {
-      console.error(error)
-      dispatch(setStatus(PROCESS_STATUSES.FAILED))
-    },
+    () => dispatch(setStatus(PROCESS_STATUSES.FAILED)),
```

Comment:
In my opinion, we should log errors. It always adds some context. We can improve it eg. `console.error("Bitcoin deposit failed", error)`